dipy/data/fetcher.py

```diff
@@ -591,6 +597,125 @@ def read_syn_data():
 """
 
 
+def fetch_viz_icons():
+    """ Download icons for visualization
+    """
+    url = 'https://dl.dropboxusercontent.com/u/2481924/'
```

Comment:
Services like that tend to come and go. Management of resources on dropbox
has a bus factor of 1. If someone else needs to deal with these files, you
have to give them your dropbox password.

On Thu, Oct 8, 2015 at 12:13 PM, Eleftherios Garyfallidis <
notifications@github.com> wrote:

> In dipy/data/fetcher.py
> <https://github.com/nipy/dipy/pull/587#discussion_r41555407>:
>
> > @@ -591,6 +597,125 @@ def read_syn_data():
> >  """
> >
> >
> > +def fetch_viz_icons():
> > +    """ Download icons for visualization
> > +    """
> > +    url = 'https://dl.dropboxusercontent.com/u/2481924/'
>
> Okay sure. But can you tell me why Dropbox is bad? I think we never had
> any problem with Dropbox. And I suppose if we move all the Dropbox files to
> NITRC we should do that all in one PR and not here. What do you think?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/dipy/pull/587/files#r41555407>.
>